clients/java/signalr/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -73,10 +72,10 @@ public HttpHubConnectionBuilder withAccessTokenProvider(Single<String> accessTok
     /**
      * Sets the duration the {@link HubConnection} should wait for a Handshake Response from the server.
      *
-     * @param timeout The duration that the {@link HubConnection} should wait for a Handshake Response from the server.
+      * @param timeout The duration (specified in milliseconds) that the {@link HubConnection} should wait for a Handshake Response from the server.
```

Comment:
nit (this one is probably my fault, GitHub was being super weird about spacing in my suggestions...): remove the extra space in front of the `*` (not going to do a suggestion for this one because it was suggestions that made this weird :))